### PR TITLE
Add support for "band" for stac conversion.

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
@alexgleith Please review.

File: libs/index/odc/index/stac.py [hunk 1]
```diff
@@ -102,10 +102,13 @@ def _get_stac_bands(
             }
 
         path = asset["href"]
+        band_index = asset["band"] if "band" in asset else None
```
Comment (alexgleith) [inline libs/index/odc/index/stac.py]:
```suggestion
        band_index = asset.get("band", None)
```
Keep it simple!

**Review by alexgleith — CHANGES_REQUESTED** [review]
If we can simplify that one line then I'm happy 🙂

File: libs/index/odc/index/stac.py (hunk 1 above)
Comment (danlipsa) [reply libs/index/odc/index/stac.py]: Thanks for the review. Yes, this makes sense. See the updated code.

**Review by alexgleith — APPROVED** [review]

**Comment by danlipsa** [comment]
Any estimate of when the pypi odc-index package will be updated? Thanks!

**Pull request merged**